databaseconnect.py

```diff
@@ -169,52 +171,55 @@ def learn_question_response(H):
     cur.execute("SELECT id FROM statement_table ORDER BY id DESC")
     res = cur.fetchone()
     last_id = res[0]
-    cur.execute(f"UPDATE statement_table SET sentence='{H}' WHERE id={last_id}")    # noqa: E501
+    cur.execute("UPDATE statement_table SET sentence = '{}' WHERE id = '{}'".format(H, last_id))  # noqa: E501
     db.commit()
     B = "Thank you! I have learnt this."
     return B, 0
 
 
+@logger_config.logger
 def clear_table(table_name):
     db = connection_to_database()
     cur = db.cursor()
 
     if table_name in ("question_table", "statement_table"):
         tables_to_be_cleaned = ("question_table", "statement_table")
-        print("The following tables will be cleaned:\n")
+        cur.execute("The following tables will be cleaned:\n")
         for table in tables_to_be_cleaned:
             describe_table(cur, table)
 
         if input("Enter 'Y' to confirm cleaning of BOTH tables: ") in ("Y", "y"):   # noqa: E501
+
             for table in tables_to_be_cleaned:
                 cur.execute(f"DELETE FROM {table}")
             db.commit()
-            print("Tables cleaned successfully")
+            cur.execute("Tables cleaned successfully")
         else:
-            print("Table cleaning skipped.")
+            cur.execute("Table cleaning skipped.")
 
     else:
-        print("The following table will be cleaned:\n")
+        cur.execute("The following table will be cleaned:\n")
         describe_table(cur, table_name)
 
         if input("Enter 'Y' to confirm: ") in ("Y", "y"):
-            print("Table cleaned successfully")
-            cur.execute(f"DELETE FROM {table_name}")
+            cur.execute("Table cleaned successfully")
+            cur.execute("DELETE FROM {table_name}")
             db.commit()
         else:
-            print("Table cleaning skipped.")
+            cur.execute("Table cleaning skipped.")
 
 
+@logger_config.logger
 def describe_table(cur, table_name):
-    cur.execute(f"DESC {table_name}")
+    cur.execute("DESC {table_name}")
     res = cur.fetchall()
     column_names = [col[0] for col in res]
 
-    cur.execute(f"SELECT COUNT(*) FROM {table_name}")
+    cur.execute("SELECT COUNT(*) FROM {table_name}")
     res = cur.fetchall()
     records_no = res[0][0]
 
-    print("Table Name:", table_name)
-    print("Columns:", column_names)
-    print("Number of existing records:", records_no)
-    print()
+    cur.execute("Table Name:", table_name)
+    cur.execute("Columns:", column_names)
+    cur.execute("Number of existing records:", records_no)
+    cur.execute()
```

Comment:
Can't replace `print` with `cur.execute`

Reply:
Oops! Did a blunder mistake in a hurry. You are right @chttrjeankr . I was intended to replace it to `logging.debug` and not `cur.execute`. I am immediately raising a fresh PR.

Reply:
Make all changes in the same PR, do not raise a new PR.